resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -189,25 +190,28 @@
     options.systemd_el = 1
   elsif options.operating_system == :el && options.os_version >= 7 # systemd el
     if ! options.is_pe
+      # https://bugzilla.redhat.com/show_bug.cgi?id=2224427
       fpm_opts << "--depends tzdata-java"
-      options.java =
-        case options.platform_version
-        when 8
-          # rpm on Redhat 7 may not support OR dependencies
-          if options.os_version == 7
-            'java-11-openjdk-headless'
-          elsif options.os_version == 8
-            '(java-17-openjdk-headless or java-11-openjdk-headless)'
-          elsif options.os_version >= 9
-            'java-17-openjdk-headless'
-          else
-            fail "Unrecognized el os version #{options.os_version}"
-          end
-        when 7
-          'java-11-openjdk-headless'
+      case options.platform_version
+      when 8
+        # rpm on Redhat 7 may not support OR dependencies
+        if options.os_version == 7
+          options.java = 'jre-11-headless'
+          options.java_bin = '/usr/lib/jvm/jre-11/bin/java'
```

Comment:
Is this (and the 17) path a stable path for all distributions of Java?

Reply:
I'm not sure it is.  On debian 12 with openjdk 17, it's `/usr/lib/jvm/java-17-openjdk-arm64/bin/java`, for example.

Reply:
Oh, but that's jdk, not jre.  Lemme try that.

Reply:
Nope, I'm wrong, that is with `openjdk-17-jre-headless` installed.